vue/assets/icons/green/icon-check-filled.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16" height="17" viewBox="0 0 16 17"><defs><path id="a" d="M8 2a6 6 0 110 12A6 6 0 018 2zm1.529 4.195L7.333 8.391l-.862-.862a.667.667 0 10-.942.942l1.333 1.334c.26.26.682.26.943 0l2.666-2.667a.667.667 0 00-.942-.943z"/></defs><g fill="none" fill-rule="evenodd" transform="translate(0 .5)"><path d="M0 0h16v16H0z"/><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><use fill="#42526E" xlink:href="#a"/><path fill="#45A777" d="M0 0h16v16H0z" mask="url(#b)"/></g></svg>
```

Comment:
Where were the icons taken from?

Reply:
@gcandal from zeplin.

Reply:
Ok. I'm sorry that I mislead you earlier when we talked about this, but didn't realise these icons were custom just for the `input` (was thinking it could be used in more components, but that's not true). Because of this, move these to the `input-platforme`-local `assets`